HTCONDOR-1659-update-quickstart (#1281)
htcondor/htcondor: docs/users-manual/quick-start-guide.rst

HTCondor Quick Start Guide
==========================

> To users, `HTCondor <http://htcondor.org>`_ is a batch job scheduler.  You give

Reviewer: Let's use https here.
```suggestion
To users, `HTCondor <https://htcondor.org>`_ is a batch job scheduler.  You give
```

    # sleep.sub -- simple sleep job

    executable              = sleep.sh

    log                     = sleep.log
    output                  = sleep.out
    error                   = sleep.err 

    should_transfer_files   = Yes
    when_to_transfer_output = ON_EXIT

>     RequestCpus             = 1
>     RequestMemory           = 512M
>     RequestDisk             = 1G

Reviewer: Let's use the condor_submit commands here
```suggestion
    request_cpus            = 1
    request_memory          = 512M
    request_disk            = 1G
```

    # sleep.sub -- simple sleep job

    executable              = sleep.bat

    log                     = sleep.log
    output                  = sleep.out
    error                   = sleep.err

    should_transfer_files   = Yes
    when_to_transfer_output = ON_EXIT

>     RequestCpus             = 1
>     RequestMemory           = 512M
>     RequestDisk             = 1G

Reviewer: ```suggestion
    request_cpus            = 1
    request_memory          = 512M
    request_disk            = 1G
```

working directory may be specified.

The ``log`` command causes a job event log file named ``sleep.log`` to be
created on the AP once the job is submitted.  A log is not
necessary, but it can be incredibly useful in figuring out what happened or
is happening with a job.

HTCondor must be told how many resources your job needs on an Execution
Point in order to run.  This allows HTCondor to run as many jobs as
possible on each EP without overloading them.  Jobs must declare the
number of CPUs, the amount of memory and disk they need.  Special jobs
may need to request other resources, such as GPUs or licenses.  Ask your
administrator if your jobs requires such things.  The amount of cpus
is unitless, but memory and disk requires can have a "M" for megabyte,
"G" for Gigabyte suffix for legibility.  Without the suffix, memory
units are megabytes and disk kilobytes.

.. code-block:: condor-submit

>     RequestCpus             = 1
>     RequestMemory           = 512M
>     RequestDisk             = 1G

Reviewer: ```suggestion
    request_cpus            = 1
    request_memory          = 512M
    request_disk            = 1G
```

direct HTCondor to explicitly send the needed files, including the executable,
to the machine where the job executes.  These commands will likely not be

> necessary for jobs in which the AP and the EP (the Execute Point, or worker
> node) machine access a shared file system.  However, including these commands

Reviewer: Let's be consistent with earlier usage
```suggestion
necessary for jobs in which the AP and the EP (the Execution Point, or worker
node) access a shared file system.  However, including these commands
```

    # science1.sub -- run one instance of science.exe
    executable              = science.exe
    arguments               = "infile-A.txt infile-B.txt outfile.txt"

    transfer_input_files    = infile-A.txt,infile-B.txt
    should_transfer_files   = IF_NEEDED
    when_to_transfer_output = ON_EXIT

>     RequestCpus             = 1
>     RequestMemory           = 512M
>     RequestDisk             = 1G

Reviewer: ```suggestion
    request_cpus            = 1
    request_memory          = 512M
    request_disk            = 1G
```

    log                     = science1.log
    queue

The input files ``infile-A.txt`` and ``infile-B.txt`` will need to be

> available on the Execution Pointer within the pool where the job

Reviewer: ```suggestion
available on the Execution Point within the pool where the job
```

    # science2.sub -- run 40 instances of science.exe
    executable              = science.exe
    arguments               = "infile-$(Process)A.txt infile-$(Process)B.txt outfile$(Process).txt"

    transfer_input_files    = infile-$(Process)A.txt,infile-$(Process)B.txt
    should_transfer_files   = IF_NEEDED
    when_to_transfer_output = ON_EXIT

>     RequestCpus             = 1
>     RequestMemory           = 512M
>     RequestDisk             = 1G

Reviewer: ```suggestion
    request_cpus            = 1
    request_memory          = 512M
    request_disk            = 1G
```

    # science3.sub -- run 100 instances of science.exe, with
    #  unique directories named by the $(Process) macro

    executable              = science.exe
    arguments               = "infile-A.txt infile-B.txt outfile.txt"

    should_transfer_files   = IF_NEEDED
    when_to_transfer_output = ON_EXIT

    initialdir              = run$(Process)
    transfer_input_files    = infile-A.txt,infile-B.txt

>     RequestCpus             = 1
>     RequestMemory           = 512M
>     RequestDisk             = 1G

Reviewer: ```suggestion
    request_cpus            = 1
    request_memory          = 512M
    request_disk            = 1G
```

directory.  This organization also works well for executables that do not
facilitate command line naming of input or output files.

> Here is a listing of the files and directories on the AP machine within

Reviewer: ```suggestion
Here is a listing of the files and directories on the AP within
```
